pysat/utils/coords.py

```diff
@@ -135,12 +135,31 @@ def calc_solar_local_time(inst, lon_name=None, slt_name='slt'):
 
 
 def scale_units(out_unit, in_unit):
-    """Deprecated function, moved to pysat.utils._core"""
+    """ Determine the scaling factor between two units
+
+    .. deprecated:: 2.2.0
```

Comment:
```suggestion
    .. deprecated:: 2.3.0
```

Reply:
The deprecation warning was in the last version.

Reply:
The file diff made it look like it wasn't.